commit c05f2d4f28e9
Author: Jonathan Kew <jkew@mozilla.com>
Date:   Wed Mar 28 14:42:20 2018 +0100

    Bug 1449157 - Guard against mContent being null in nsFrame::HandlePress. r=dholbert, a=jcristau
    
    --HG--
    extra : rebase_source : cee8a6f6ecce503cddf3492ffaf42bc317f1851c
    extra : source : c09b8a694cb5d0442aa293eace75a18058b675d5
---
 layout/generic/nsFrame.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git layout/generic/nsFrame.cpp layout/generic/nsFrame.cpp
index fd313f9a792f..5fa8bd9174e5 100644
--- layout/generic/nsFrame.cpp
+++ layout/generic/nsFrame.cpp
@@ -4275,7 +4275,7 @@ nsFrame::HandlePress(nsPresContext* aPresContext,
   // starting a new selection since the user may be trying to
   // drag the selected region to some other app.
 
-  if (GetContent()->IsSelectionDescendant())
+  if (GetContent() && GetContent()->IsSelectionDescendant())
   {
     bool inSelection = false;
     UniquePtr<SelectionDetails> details
